releasenotes/notes/48833.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: release-notes/v2
+kind: feature
+area: telemetry
+releaseNotes:
+  - |
+    **Added** support for disable bootstrap tracing configuration, this feature is disabled by default and can be
+    enabled by setting the `DISABLE_BOOTSTRAP_TRACING` environment variable in istiod and proxy.
```

Comment:
Add a reason why someone would want to do it?

Reply:
The main reason is that you don't have to restart proxies to change tracing.